llvm/lib/Target/AMDGPU/VOPCInstructions.td

```diff
@@ -1131,16 +1131,16 @@ defm : ICMP_Pattern <COND_SLT, V_CMP_LT_I64_e64, i64>;
 defm : ICMP_Pattern <COND_SLE, V_CMP_LE_I64_e64, i64>;
 
 let OtherPredicates = [HasTrue16BitInsts] in {
```

Comment:
Should be True16Predicate = UseFakeTrue16Insts ?

Reply:
Updated. I was thinking about updating this in the later patch but we can do this here as well